src/documenti/esterni/AdR/contenuti/requisiti.tex

```diff
@@ -1,96 +1,224 @@
 \chapter{Requisiti}
 
+\renewcommand\arraystretch{1,5}
+
 \section{Introduzione}
 \section{Requisiti Funzionali}
-\begin{table}[H]
+
+\begin{center}
   \centering
-  \begin{tabular}{|c|c|c|c|}
+  \begin{longtable}{|L{1,5cm}|C{5,5cm}|C{3cm}|C{2cm}|}
     \hline
     \rowcolor[HTML]{036400}
-    {\color[HTML]{FFFFFF} \textbf{Codice}} & {\color[HTML]{FFFFFF} \textbf{Classificazione}} & {\color[HTML]{FFFFFF} \textbf{Descrizione}} & {\color[HTML]{FFFFFF} \textbf{Fonti}} \\ \hline
+    \textcolor[HTML]{FFFFFF}{\textbf{Codice}} & \textcolor[HTML]{FFFFFF}{\textbf{Descrizione}} & \textcolor[HTML]{FFFFFF}{\textbf{Classificazione}} & \textcolor[HTML]{FFFFFF}{\textbf{Fonti}}
+    \\ \hline
+    \rowcolor[HTML]{EFEFEF}
+    RF.1.1 & L'utente deve poter caricare i dati tramite un nuovo dataset & Obbligatorio & Capitolato - UC1 \\ \hline
+    \rowcolor[HTML]{C0C0C0}
+    RF.1.2 & Visualizzazione messaggio di errore in caso di problemi durante il caricamento dati & Obbligatorio & UC2 \\ \hline
+    \rowcolor[HTML]{EFEFEF}
+    RF.2.3 & L'utente deve poter caricare una sessione precedentemente salvata & Desiderabile & UC3 \\ \hline
+    \rowcolor[HTML]{C0C0C0}
+    RF.1.4 & Visualizzazione messaggio di errore in caso di problemi durante il caricamento della sessione & Obbligatorio & UC4 \\ \hline
+    \rowcolor[HTML]{EFEFEF}
+    RF.1.5 & L'utente deve poter selezionare le dimensioni da visualizzare nel grafico & Obbligatorio & UC5 \\ \hline
+    \rowcolor[HTML]{C0C0C0}
+    RF.1.6 & L'utente deve poter selezionare il grafico da visualizzare & Obbligatorio & Capitolato - UC6 \\ \hline
+    \rowcolor[HTML]{EFEFEF}
+    RF.1.6.1 & L'utente deve poter selezionare il grafico \textit{Scatter Plot} & Obbligatorio & Capitolato - UC6.1 \\ \hline
+    \rowcolor[HTML]{C0C0C0}
+    RF.1.6.2 & L'utente deve poter selezionare il grafico \textit{Parallel Coordinates} & Obbligatorio & Capitolato - UC6.2 \\ \hline
     \rowcolor[HTML]{EFEFEF}
-    &  &  &  \\ \hline
+    RF.1.6.3 & L'utente deve poter selezionare il grafico \textit{Force-Direct Graph} & Obbligatorio & Capitolato - UC6.3 \\ \hline
     \rowcolor[HTML]{C0C0C0}
-    &  &  &  \\ \hline
-  \end{tabular}
-  \caption{Tabella dei requisiti funzionali}
-\end{table}
+    RF.1.6.4 & L'utente deve poter selezionare il grafico \textit{Sankey Diagram} & Obbligatorio & Capitolato - UC6.4 \\ \hline
+    \rowcolor[HTML]{EFEFEF}
+    RF.2.7 & L'utente deve poter personalizzare la visualizzazione del grafico & Desiderabile & UC7 \\ \hline
```

Comment:
Forse qua sarà il caso di aggiungere qualche requisito più specifico in base a come hanno fatto il poc

Reply:
Si potrebbe starci, poi eventualmente se quelli del poc si sono resi conto di requisiti aggiuntivi o requisiti non possibili si andranno a modificare